packages/react-storage/src/components/StorageBrowser/views/hooks/__tests__/useSearch.spec.ts

```diff
@@ -0,0 +1,113 @@
+import { renderHook, act } from '@testing-library/react-hooks';
```

Comment:
We've been trying to get away from using this package and using `'@testing-library/react'` instead (even though it will tell you `act` is deprecated 😅 )